app/controllers/users.js

```diff
@@ -91,6 +91,14 @@ exports.create = function (req, res) {
   });
 };
 
+exports.changePassword = function (req, res) {
+  var user = req.profile;
```

Comment:
@mahmoudmahdi24 Why do you need this variable. This function just create a variable? You are sure that you commited all your changes?